src/clj/api/resolvers.clj

```diff
@@ -121,3 +121,11 @@
     (clj->gql {:total-count total-count
                :edges       edges'
                :page-info   page-info})))
+
+(defn analysis->maps [_ _ _]
+  #_[{:keys [db]} _ {analysis-id :id :as parent}]
+  ;; TODO: implement this once we have refactored the db and added bounding boxes to the analysis parent table
```

Comment:
I have no problem storing the bounding box in the DB, but we also discussed that the SpreadData json file can be self-contained and the bounding box can be obtained by following the AxisAttribute.x / AxisAttribute.y pointers to the Attributes list and reading the range field that they have. ContinuousTree parser definitely has this data and we can make sure that the other analysis parsers store it to. Your call, I can add this information to the other Java parsers in a next PR.

Reply:
This I'll take a look at what it takes to grab that bounding box from the analysis data and if it is super easy we can go with that, without adding it to the db